src/Firehose.Web/Firehose.Web.csproj

```diff
@@ -20,6 +20,7 @@
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
     <UseGlobalApplicationHostFile />
+    <Use64BitIISExpress />
```

Comment:
Don't change this

Reply:
I guess you only need to remove this now @balivo